src/util.js

```diff
@@ -24,6 +24,20 @@ var Util = {
         total: 0,
         message: null
     },
+    
+    countOccurences: function(contentString, searchString) {
+        var count = 0;
+
+        var searchRegex = new RegExp(searchString, "g");
+        var widgetRegex = /^\[\[\u2603 (([a-z-]+) ([0-9]+))\]\]/g;
+
+        var matches = contentString.replace(widgetRegex, "").match(searchRegex);
+        if (matches !== null) {
+            count += matches.length;
```

Comment:
This was left over from a previous revision where I was counting occurrences in ```question.content``` and each of the hints.  It no longer makes sense.  It will be completely replaced when the changes described above land.